poppy/utils.py

```diff
@@ -373,8 +388,18 @@ def display_profiles(HDUlist_or_filename=None,ext=0, overplot=False, title=None,
         HDUlist = HDUlist_or_filename
     else: raise ValueError("input must be a filename or HDUlist")
 
+    im = HDUlist[ext].data.copy() # don't change normalization of actual input array, work with a copy!
+
+    if normalize.lower() == 'peak':
+        _log.debug("Displaying PSF normalized to peak = 1")
+        im /= im.max()
+    elif normalize.lower() =='total':
+        _log.debug("Displaying PSF normalized to PSF total = 1")
+        im /= im.sum()
+        
+    HDUlist_copy = fits.HDUList([fits.PrimaryHDU(data=im,header=HDUlist[ext].header)])
```

Comment:
Hmm, the fact that the exact same code is needed in two places suggests maybe this normalization code belongs inside `radial_profile()` itself.  Then `display_EE` and `display_profiles` could both just pass the normalization parameter through as part of the kwargs.